go/server.go

```diff
@@ -124,14 +129,22 @@ func setPaymentToken(c *gin.Context) {
 	fmt.Println("payment token: " + paymentToken)
 	fmt.Println("payment id: " + paymentID)
 
+	linkToken, httpErr := fetchLinkToken(paymentID)
+	if httpErr != nil {
+		c.JSON(httpErr.errorCode, gin.H{"error": httpErr.Error()})
+	}
 	c.JSON(http.StatusOK, gin.H{
-		"payment_token": paymentToken,
```

Comment:
we should still return the payment token here for the front-end to work right?

Reply:
Nope! The link_token now replaces the payment_token :)

Reply:
nice, we'll need to update the front-end in that case to use the link token instead of payment token i believe

Reply:
this is purely for display purposes